test/test_client.py

```diff
@@ -510,15 +512,18 @@ def test_wait_as_completion_timeouted(self, timeout=0):
         start = 1
         data = bytearray(size)
         self.client.write_area(area, dbnumber, start, data)
-        # start as_request and wait for zero seconds to trigger timeout
-        self.client.as_db_read(dbnumber, start, size)
-        try:
-            self.client.wait_as_completion(ctypes.c_ulong(0))
-        except Snap7Exception as s7_err:
-            if not s7_err.args[0] == b'CLI : Job Timeout':
-                self.fail(f"While waiting another error appeared: {s7_err}")
-        except BaseException as pyt_err:
-            self.fail(f"Exception was thrown: {pyt_err}")
+        # start as_request and wait for zero seconds to try trigger timeout
+        for i in range(tries):
```

Comment:
I noticed in my tests, that sometimes a zero second timeout is not fast enough...So I added a loop with tries (in latest commit 100), to evoke this timeout error.
If this is not happening, the test should be skipped